File: Baekjoon_Online_Judge/NO_15964/15964.c
#include <stdio.h>
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 07                   | 
 | 15964 이상한 기호          | 
 ---------------------------- 
*/

int main(){
    long long A, B;

    scanf("%lld %lld", &A, &B);
    printf("%lld", (A + B) * (A - B));
    return 0;
}

File: Baekjoon_Online_Judge/NO_15964/15964.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 07                   | #
# | 15964 이상한 기호         | #
# ---------------------------- #


if __name__ == "__main__":
    A, B = map(int, r_input().split())

    print((A + B) * (A - B))
